grails-web-boot/src/test/groovy/grails/boot/EmbeddedContainerWithGrailsSpec.groovy

```diff
@@ -32,8 +31,7 @@ class EmbeddedContainerWithGrailsSpec extends Specification {
             new URL("http://localhost:${context.webServer.port}/foos").text == 'all foos'
     }
 
-    @Configuration
-    @EnableWebMvc
+    @SpringBootApplication
```

Comment:
Does this mean any GrailsApplication will require this going forward or is this because this code is in a test environment?

Reply:
@jdaugherty @EnableAutoConfiguriation is what is needed. This just simplifies to 1 annotation.